queries.sql

```diff
@@ -7,3 +7,60 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name<>'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Task 2 queries.
+-- No 1.
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+-- No 2.
+BEGIN;
+
+UPDATE animals 
+SET species = 'digimon'
+WHERE name like '%mon';
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species = '';
+
+COMMIT;
+SELECT * FROM animals;
+
+--No 3.
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+--No 4.
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SELECT * FROM animals;
+SAVEPOINT checkpoint1;
+UPDATE animals SET weight_kg=weight_kg*-1;
+SELECT * FROM animals;
+ROLLBACK TO checkpoint1;
+UPDATE animals SET weight_kg=weight_kg*-1 where weight_kg<0;
+SELECT * FROM animals;
+COMMIT;
+
+--No 5
+SELECT COUNT(*) AS total FROM animals;
+
+SELECT COUNT(*) AS coolAnimals FROM animals
+WHERE escape_attempts=0;
+
+SELECT AVG(weight_kg) AS meanWeight FROM animals;
+
+SELECT neutered, SUM(escape_attempts) AS most_escape_attempts FROM animals GROUP BY neutered;
+
```

Comment:
- [ ] I think it would be better if you update this `query` in order to correctly answer the asked question. You are expected to get **the animal who escapes the most, neutered or not neutered** and not **the sum of escape_attempts**. 
I suggest you update your query by using `MAX` instead of `SUM` and add the screenshot of your result in the pull request. I know you can do that :muscles: :muscles: